### ISSUE TITLE
`RefuseManualStop` can be bypassed with `service` command

### ISSUE THREAD

**Issue by nisbet-hubbard** [systemd/systemd#34288]
### systemd version the issue has been seen with

systemd-252-32.el9_4.6

### Used distribution

RHEL 9

### Linux kernel version used

5.14.0-427.31.1.el9_4.x86_64

### CPU architectures issue was seen on

x86_64

### Component

systemctl

### Expected behaviour you didn't see

`Failed to restart auditd.service: Operation refused …`

### Unexpected behaviour you saw

```bash
Stopping logging: 
Redirecting start to /bin/systemctl start auditd.service
```

### Steps to reproduce the problem

`service auditd restart`

### Additional program output to the terminal or log subsystem illustrating the issue

```sh
Sep  6 17:28:45 systemd[1]: Reloading.
Sep  6 17:28:45 systemd-rc-local-generator: /etc/rc.d/rc.local is not marked executable, skipping.
Sep  6 17:29:29 sedispatch[640]: sedispatch is exiting on stop request
Sep  6 17:29:29 auditd[636]: The audit daemon is exiting.
Sep  6 17:29:29 kernel: audit: type=1305 audit(1725614969.154:635355): op=set audit_pid=0 old=636 auid=4294967295 ses=4294967295 subj=system_u:system_r:auditd_t:s0 res=1
Sep  6 17:29:29 systemd[1]: auditd.service: Deactivated successfully.Sep  6 17:29:29 systemd[1]: auditd.service: Consumed 1min 16.337s CPU time.
Sep  6 17:29:29 kernel: audit: type=1131 audit(1725614969.157:635356): pid=1 uid=0 auid=4294967295 ses=4294967295 subj=system_u:system_r:init_t:s0 msg='unit=auditd comm="systemd" exe="/usr/lib/systemd/systemd" hostname=? addr=? terminal=? res=success'
Sep  6 17:29:31 systemd[1]: Starting Security Auditing Service...
Sep  6 17:29:31 auditd[378488]: audit dispatcher initialized with q_depth=2000 and 1 active plugins
Sep  6 17:29:31 auditd[378488]: Init complete, auditd 3.1.2 listening for events (startup state enable)
Sep  6 17:29:31 augenrules[378493]: /sbin/augenrules: No change
```

**Comment by YHNdnzj** [comment]
`service` is not a systemd command, but most likely some compat glue provided by your distro. My guess is that when stopping service it bypasses systemd and invokes the corresponding rc.d script. Hence this should really be reported to your distro instead, or please just use systemctl.

**Comment by poettering** [comment]
you can trivially circumvent RefuseManualStop=, it's not a security feature, but a robustness feature. (i.e. you can just start a service that has a dep on such a service to start it, i.e. systemd-run for example)

I have no idea what "service" internally does, it's not a systemd thing. It's 2024 after all, sysvinit is long dead.

Also v252 is *waaaaaaaaaay* too old to be tracked here. This is an upstream systemd tracker.

Long story short: please contact your OS/distro vendor instead.

**Comment by poettering** [comment]
/
*[Canned reply follows]*

This is the upstream bug and feature request tracker of systemd. Please use this only for issues in the two most current upstream systemd versions. See this link for the list of current releases.

https://github.com/systemd/systemd/releases

For support for older versions please contact your distribution instead.

In order to keep frustration at a minimum the submission form tried to make this all very clear.
